src/com/palmergames/bukkit/towny/TownyFormatter.java

```diff
@@ -195,47 +195,52 @@ public static String getTime() {
 			}
 		out.add(line);
 		

```

Comment:
So what I think would make this a lot cleaner is if at this point, instead of nesting everthing in a `if (!resident.isNPC())`, instead you can just put this on this line:
```
		if (resident.isNPC()) {
			out.add(Translation.of("msg_status_npc", resident.getName()));
			out.addAll(getExtraFields(resident));
			out = formatStatusScreens(out);
			return out;
		}
```

Reply:
Also all of the changes above and below this line will have to be un-done.

Reply:
Done